### ISSUE TITLE
Task timeout

### ISSUE THREAD

**Issue by boite** [droid-php/droid#10]
By default, Symfony Process sets an upper bound on process execution time: `$timeout` is set to 60s.  Exceptions thrown because the runtime of a task exceeded this limit were encountered while testing modules, particularly when installing platform packages with apt-get.

Most tasks should complete in well under 60s, but it should be possible for a Task to declare a longer max execution time.

Symfony Process also allows for unlimited execution time and it might be useful to allow Tasks to run unlimited too - I can't immediately think of a use case for this.

Suggested solution:

```
Tasks:
  - name: "Do a thing for a long time."
    command: "some:command"
    max_runtime: 300 # seconds; or 0 for unlimited; not set == default (60s)
```

**Comment by joostfaassen** [comment]
@boite adding a max_runtime option to all commands is a bit messy. I'm thinking we should disable timeouts all together in all Symfony Process runs, the 60 second limit is pretty arbitrary. If we'd want to implement time-outs properly, we should allow to specify them at the task level, and the task runner should keep an eye on if the task is running too long. It shouldn't matter if this is a regular php command or something spawned by Symfony Process.
What do you think?
For the moment I'll just disable the timeout for the apt commands.

**Comment by boite** [comment]
@joostfaassen there's two things that need to be done then:-
1. Set a reasonable default `Task::RUNTIME_MAX` (current value is 60s) - the TaskRunner is already using Task.maxRuntime to keep an eye on run time.
2. Go through all existing commands which use SymfonyProcess and disable the timeout.

**Comment by boite** [comment]
Fixed in v1.19.0
